benchmarks/yeps.js

```diff
@@ -0,0 +1,11 @@
+const http = require('http')
+const App = require('yeps')
+
+const app = new App()
+
+app.then(async ctx => {
+  ctx.res.statusCode = 200
+  ctx.res.end('{"hello":"world"}')
```

Comment:
Can you please use `JSON.stringify` here?

Also, you'll need to set the proper header like in https://github.com/fastify/benchmarks/blob/1e6d459dad67ca7f957d086f9cb9830b14533d44/benchmarks/bare.js#L4-L5.